Makefile

```diff
@@ -55,7 +55,7 @@ ifneq ($$(__dep_$1.$2),)
 endif
 
 .$1.$2.stamp: $1.Dockerfile $(wildcard docker/*)
-	./build-img $(if $(TAG),-V "$(TAG)") "$(DOCKER_ORG)/$1:$2-$(docker_tag)"
+	./build-img $(if $(TAG),-V "$(TAG)") "$(DOCKER_ORG)/$1:$2-$(docker_tag)" $(BUILD_ARGS) $($1.$2.BUILD_ARGS)
```

Comment:
Quotes around `$(BUILD_ARGS)` and `$($1.$2.$BUILD_ARGS)`

Reply:
I want no quotes, otherwise we can't pass multiple arguments :)

Reply:
Will the `$1.$2. ` work in that case?

Reply:
Ah no ignore me.

Reply:
(You're building the name of the env variable, not the arguments).